extensions/control-plane/provision/provision-oauth2/docs/diagram.dot

```diff
@@ -0,0 +1,31 @@
+digraph {node [ shape=square ];
```

Comment:
That's because I'm not familiar with plantuml :smiling_face_with_tear:  I ended up keeping only the png file, that will be just enough for the documentation.

Reply:
It is not difficult to get into, and there's a serviceable plugin for IntelliJ with live updates and everything.